cgsmiles/drawing.py

```diff
@@ -43,15 +43,20 @@ def draw_molecule(graph,
                   colors=None,
                   labels=None,
                   scale=2,
-                  outline=False,
+                  outline=None,
                   use_weights=False,
                   align_with='diag',
                   fontsize=10,
                   text_color='black',
                   edge_widths=3,
+                  edge_spacing=0.1,
+                  edge_separation=0.2,
                   mapped_edge_width=20,
                   default_bond=1,
-                  layout_kwargs={}):
+                  radius=None,
+                  piezorder=3,
```

Comment:
Does this need to be exposed?

Reply:
this is just a play branch to see some options for making differently styled images. They are all worse than the default though